app/core/models.py

```diff
@@ -182,7 +182,10 @@ class SponsorPartner(AbstractBaseModel):
     is_org_sponsor = models.BooleanField(null=True)
     is_org_partner = models.BooleanField(null=True)
 
-    # PK of this model is the ForeignKey for project_partner_xref
+    # PK of this model is the ForeignKey for project_affiliate_xref
+
+    class Meta:
+        db_table = "affiliate"
```

Comment:
I actually meant to change both the model name and the table name to `affiliate`.